### DISCUSSION TITLE
Restart ODK server

### DISCUSSION THREAD

**Discussion by Nik720** [Samagra-Development/SamagraX-community#5] [Installation and Configuration]
@ChakshuGautam   @SaketSinha I have updated the ODK form and it is not reflecting in the WhatsApp bot https://agg.staging.saksham.samagra.io/Aggregate.html#management/forms/
requesting you to please restart the server as mentioned in [document](https://docs.google.com/document/d/1NG2KrXwHl6vHfVz4e-niSK0PhB1UHIHkUlvWMwtYIoo/edit#)

CC @vinukumar-vs @vrayulu

**Comment by SaketSinha** [comment]
@Nik720 - This is done. Please check

cc- @aayushkedia95

**Reply by Nik720** [reply]
@SaketSinha It is working now. Thanks

cc - @vinukumar-vs @vrayulu

**Reply by Nik720** [reply]
@ChakshuGautam @SaketSinha I have updated the ODK form in ODK server https://agg.staging.saksham.samagra.io/Aggregate.html#management/forms/
requesting you to please restart the server as it is not reflecting in the WhatsApp bot

CC @vinukumar-vs @vrayulu

**Reply by aayushkedia95** [reply]
Getting this done, @Nik720.

**Reply by aayushkedia95** [reply]
@Nik720, this was done. Please check and confirm.

**Reply by Nik720** [reply]
@aayushkedia95 it is working. Thanks

**Reply by Nik720** [reply]
@ChakshuGautam @SaketSinha I have updated the ODK form in the ODK server, requesting you to please restart the server

cc -  @aayushkedia95 @vinukumar-vs

**Reply by aayushkedia95** [reply]
@Nik720, restarted, please check.

**Reply by Nik720** [reply]
@aayushkedia95 After updating the form I am getting  **Invalid Input!!! Please try again.** as a response after selecting any option. I have checked the workflow with ODK collect app and there it is working fine. 

FormID: sunbird_usecases_05_april
conversation logic id: d0d98788-6a69-44b6-81e0-a8cd83aec4bb
bot id: 25504ebc-fb84-4bc4-8907-76fa12dc77f8
form URL: https://drive.google.com/file/d/1dKjmNx596QKzMAOgde67DDhMnzmr1eBJ/view?usp=sharing

<img width="300" alt="Screenshot 2021-04-05 at 3 25 47 PM" src="https://user-images.githubusercontent.com/33539271/113562187-4a546a00-9623-11eb-928d-81cb44b7a8ed.png">

**Reply by aayushkedia95** [reply]
@Nik720, checking.

**Reply by aayushkedia95** [reply]
Hey @Nik720, Because the labels that you have in the choices sheet have a dot after the number - '1. Digital Content', if you enter '1.' or '2.', it's working fine. 

It's a small bug in the system, we'll be resolving it by tomorrow afternoon. Post that both '1' and '1.' would work. 

![Screenshot 2021-04-05 at 5 23 39 PM](https://user-images.githubusercontent.com/80503666/113571321-cf477f80-9633-11eb-95b4-33d97f46e080.png)

**Reply by Nik720** [reply]
Thanks @aayushkedia95

**Reply by Nik720** [reply]
@ChakshuGautam @SaketSinha I have updated the ODK form in the ODK server, requesting you to please restart the server

cc - @aayushkedia95 @vinukumar-vs

**Reply by aayushkedia95** [reply]
@Nik720, done. Please check.

**Reply by Nik720** [reply]
@aayushkedia95 It is working. Thanks

**Reply by Nik720** [reply]
@aayushkedia95 @ChakshuGautam I am not getting a response from the WhatsApp samagra bot can someone please check? 

FormID: sunbird_usecases_05_april
conversation logic id: d0d98788-6a69-44b6-81e0-a8cd83aec4bb
bot id: 25504ebc-fb84-4bc4-8907-76fa12dc77f8
form URL: https://drive.google.com/file/d/1dKjmNx596QKzMAOgde67DDhMnzmr1eBJ/view?usp=sharing

<img width="300" alt="Screenshot 2021-04-20 at 10 38 51 AM" src="https://user-images.githubusercontent.com/33539271/115340805-0b96e600-a1c5-11eb-873d-72ec093d8dbc.png">

cc - @vinukumar-vs @vrayulu

**Reply by Nik720** [reply]
@aayushkedia95 @ChakshuGautam Any update on the above? This is a blocker for us to show a demo

cc - @vinukumar-vs @vrayulu

**Reply by aayushkedia95** [reply]
Hi @Nik720, we are looking into this issue. Please allow us sometime.

**Reply by aayushkedia95** [reply]
@Nik720, please check now. It has started working. 

We had switched off the sandbox for upgradation, and forgot to switch it on. It's done now. Please confirm.

**Reply by Nik720** [reply]
@aayushkedia95 It is working now. Thanks

**Reply by Nik720** [reply]
@ChakshuGautam I have updated the ODK form in the ODK server and created a new Bot, requesting you to please restart the server

cc - @aayushkedia95 @vinukumar-vs

**Reply by aayushkedia95** [reply]
Can you check now @Nik720?

**Reply by Nik720** [reply]
@ChakshuGautam I have updated the ODK form in the ODK server. requesting you to please restart the server

cc - @aayushkedia95 @vinukumar-vs

**Reply by Nik720** [reply]
@aayushkedia95 Any update on the above request?

cc - @vinukumar-vs

**Reply by aayushkedia95** [reply]
Hey @Nik720, can you keep the form name less than 20 characters? That is creating an issue. Let me know once done.

**Reply by Nik720** [reply]
@aayushkedia95 I have updated it

**Reply by aayushkedia95** [reply]
Can you share the form name? @Nik720

**Reply by Nik720** [reply]
Form title: Sunbird Tara V5
Form ID: sunbird_tara_v5

**Reply by aayushkedia95** [reply]
Please check now, @Nik720

**Reply by Nik720** [reply]
@aayushkedia95  can you please restart it again, I have created a new bot

**Reply by aayushkedia95** [reply]
Done @Nik720

**Reply by Nik720** [reply]
@aayushkedia95 I have made changes in ODK form, requesting you to please restart the server 

cc - @vinukumar-vs

**Reply by aayushkedia95** [reply]
Done @Nik720
